packages/containers/src/FilterBar/FilterBar.container.js

```diff
@@ -67,8 +67,8 @@ class FilterBar extends React.Component {
 	render() {
 		const state = this.props.state || DEFAULT_STATE;
 		const props = Object.assign({}, omit(this.props, cmfConnect.INJECTED_PROPS), {
-			docked: state.get('docked'),
-			value: state.get(QUERY_ATTR, ''),
+			docked: this.props.docked !== undefined ? this.props.docked : state.get('docked'),
```

Comment:
You should use `this.props.docked != null` to handle `null` and `undefined` (and add `const DOCKED_ATTR = 'docked';` at the top of this file :p )